swift/swift.py

```diff
@@ -141,19 +146,21 @@ def createApp(self, name: str, info: AppInfo):
         else:
             app = cls(name, parent=self)
         app.broadcastRequested.connect(self._broadcast, type=Qt.QueuedConnection)
+        app.swiftcallRequested.connect(self._callSwift, type=Qt.QueuedConnection)

```

Comment:
How about `self._swiftcall()`? I think `_callSwift()` looks weird.

Reply:
I applied it! Please check it.